src/System.Security.Permissions/src/System/Security/Permissions/ResourcePermissionBase.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace System.Security.Permissions
 {
+    [Serializable]
```

Comment:
Thinking about it again... When I ported the code I might removed all Serializable attributes intentionally because they are all stubs and don't need to be serializable at all (at least I think so).